docs/hints-for-use-with-docker.md

```diff
@@ -0,0 +1,53 @@
+# Hints for using ORT with Docker
+
+## General Hints
+
+### Mount current working directory or directory relative to working directory
+
+```shell
+docker run \
+  -v $PWD/:/project  \ # Mount current working directory into /project to use as input.
+  -v $PWD.ort:/ort \ # Mount folder "<workingdirectory>.ort" to /ort to use as output and config file storage.
+  ort --info analyze \
+  -c /ort/config.hocon \ # Use file from "<workingdirectory>.ort" as config.
+  analyze (...) # Insert further arguments for the command.
+```
+
+**Note:** The single forward slash `/` between the environment variable `$PWD` and the `:` required for the Docker `-v` argument is required for PowerShell compatibility, as PowerShell otherwise interprets `:` as part of the environment variable. 
```

Comment:
I guess the "required for the Docker `-v` argument " part should simply be omitted?